distribution/docs/src/main/resources/templates/documentation.ftl

```diff
@@ -35,10 +35,10 @@ include::${project.build.directory}/doc-contents/content/config.adoc[]
 == Eventing
 
 <#include "eventing.ftl">
-
+<#-- 
```

Comment:
Is this suppose to be commented out. If so can we just remove it?

Reply:
fixed.